tests/test-cases/Group1-Docker-Commands/1-19-Docker-Volume-Create.robot

```diff
@@ -82,6 +82,28 @@ Docker volume create with specific capacity
     Should Be Equal As Integers  ${rc}  0
     Should Be Equal As Strings  ${disk-size}  96.0G
 
+Docker volume create large volume specifying units
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create --name=unitVol1 --opt Capacity=10G
+    Should Be Equal As Integers  ${rc}  0
+    Should Be Equal As Strings  ${output}  unitVol1
+    Set Suite Variable  ${ContainerName}  unitContainer
+    Run  docker ${params} run --name ${ContainerName} -d -v ${output}:/mydata busybox /bin/df -Ph
```

Comment:
We should grab the exit code and output here, and check that the exit code is 0.